test/test_bsonjs.py

```diff
@@ -143,11 +152,6 @@ def test_regex(self):
         res = self.round_tripped({"r": regex})["r"]
         self.assertEqual(unicode_options, res.flags)
 
-        # Some tools may not add $options if no flags are set.
-        # https://jira.mongodb.org/browse/CDRIVER-3773
-        self.assertRaises(ValueError, bsonjs_loads, '{"r": {"$regex": '
-                                                  '"a*b"}}')
```

Comment:
What happens when parsing this string now? We should keep the test for this case.

Reply:
Done.